android/build.gradle

```diff
@@ -1,12 +1,17 @@
 apply plugin: 'com.android.library'
 
+def DEFAULT_COMPILE_SDK_VERSION             = 26
```

Comment:
can you change it back to 25? inline with react-native default

Reply:
Alright, I will do it. Any other requests?

Reply:
that's it

Reply:
Done, thanks @evollu for reviewing in such short notice. Might want to update the readme to have a section where users can define their own Google Play Services version as per outlined in Android documentation (https://developer.android.com/studio/build/gradle-tips.html#configure-project-wide-properties)